### PR TITLE
Show user's location on map. Bump map API version to 3.58.

### PR DESCRIPTION
**Pull request by wcruppel**
Nothing more to explain, was pretty simple.

BTW, there is a warning in the Console about the Marker class being deprecated and replaced by AdvancedMarkerElement. I took a quick look, but it's not a simple as dropping in the new class.

### CODE REVIEW

**Comment by nomatteus** [comment]
Thanks for the PR! I've deployed this change to the main site at https://totransit.ca/, and will test it out a bit before merging. Will let you know if I have any additional feedback.

File: js/ttc.js [hunk 1]
```diff
@@ -473,13 +483,27 @@ var Controls = (function() {
     	torontoBoundsNE = new google.maps.LatLng(43.930, -79.095),
     	torontoBounds = new google.maps.LatLngBounds(torontoBoundsSW, torontoBoundsNE);
 
+	 // Create a marker for the user's location
+	 window.userloc = new google.maps.Marker({
+		clickable: false,
+		icon: new google.maps.MarkerImage (
+			'//maps.gstatic.com/mapfiles/mobile/mobileimgs2.png',
```
Comment (nomatteus) [inline js/ttc.js:490]: How did you find out about this image file? Is there a reference somewhere? (Wondering if there's any other options for styles)

**Review by nomatteus — APPROVED** [review]
Looks great, tested this out on the go and it's very helpful to see your current location on the map. Thanks for making this change!

File: js/ttc.js (hunk 1 above)
Comment (wcruppel) [reply js/ttc.js:490]: From some sample code I found. Unfortunately, Google does not have an official list of their maps icons (or even the API used to request them), and they keep changing the way they are requested. Although, at least they seem to keep the legacy methods around too.

From looking at the requests Maps makes via browser Dev Tools, here's an example for the location pin:

https://www.google.com/maps/vt/icon/name=assets/icons/poi/quantum/modern_pinlet/dot_pinlet-2-medium.png&scale=1&highlight=0000FF

The scale param is a real number (e.g. 1.75 works), and highlight is the RGB in hex.

**Comment by wcruppel** [comment]
You're welcome. Going to a Leafs game this Wed, so looking forward to trying it out! Cheers.

**Pull request merged**